src/org/plovr/ConfigOption.java

```diff
@@ -272,6 +272,16 @@ public void apply(String outputCharset, Config.Builder builder) {
     }
   }),
 
+
+  CACHE_OUTPUT_FILE("cache-output-file", new ConfigUpdater() {
```

Comment:
I'm fine making it the default behavior in which case there could be a opt-out option to disable it.  I made it opt-in in the case there were unintended side-effects, but I don't see any at this time.